pages/common/bpkg.md

```diff
@@ -0,0 +1,28 @@
+# bpkg
+
+> A package manager for Bash scripts.
+> More information: <https://github.com/bpkg/bpkg>.
+
+- Update the local index:
+
+`bpkg update`
+
+- Install a package [g]lobally:
```

Comment:
```suggestion
- Install a package globally:
```

Short options only require `[]` when possible. `--global` is not a short option.

Reply:
> Short options only require [] when possible. --global is not a short option.

We allow mnemonics to be associated with words explaining to context too.

Reply:
This was discussed in a previous PR. Let me find it and update this comment with the link.

Reply:
> This was discussed in a previous PR. Let me find it and update this comment with the link.

Yeah, I recollected the discussion about it; and applied your suggestion.